pallets/token-manager/src/test_non_avt_tokens.rs

```diff
@@ -234,14 +234,6 @@ fn avn_test_signed_transfer_with_valid_input_should_succeed() {
             <TokenManager as Store>::Balances::get((NON_AVT_TOKEN_ID_2, recipient_account_id)),
             4 * amount
         );
-
-        assert!(System::events().iter().any(|a| a.event ==
-            RuntimeEvent::TokenManager(crate::Event::<TestRuntime>::TokenTransferred {
-                token_id: NON_AVT_TOKEN_ID,
-                sender: sender_account_id,
-                recipient: recipient_account_id,
-                token_balance: amount
-            })));
```

Comment:
We should be changing only the logic for AVT transfers. That means the tests for NON-AVT tokens should not be touched.